src/isPalindrome.js

```diff
@@ -0,0 +1,17 @@
+export default function isPalindrome(string) {
+  // let regex = (/^[0-9.,:;]+$/)
+  let regex = /[\W_]/g
+  let check = string.toLowerCase().replace( regex, '' )
+  let emptyArray = ''
+
+  for( let i = check.length - 1; i >= 0; i-- ) {
+      emptyArray += check[i]
+  }
+
+  if (check === emptyArray ) {
```

Comment:
this code feels like it could be refactored.  An if statement that just returns true/false could at least bea ternary, but you could probably write this in a way that collapses this and the final return??